GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -10,11 +10,39 @@ import Foundation
 import MobileCoreServices
 import Photos
 
-internal final class FilePickerManager: NSObject {
+protocol DocumentPickerCoordinatorDelegate: class {
+    /**
+     Called when a user pick one or several files from both the gallery or files explorer.
+     The completion might provide with errors than must be handled here before dismissing the
```

Comment:
The completion might provide <strike>with</strike> errors tha**t** must be handled here before dismissing the